### ISSUE TITLE
Different Card Views and Cleaning Up UXUI

### ISSUE THREAD

**Issue by avo-ucsd** [cse110-sp25-group8/final-project#42]
## [WHY DO WE HAVE THIS FEATURE?]
Different card views are an accessibility focused feature. Users should be able to set what preferred view they want.

## [TASK DESCRIPTION]
There are two goals to this: help out the other teams with UI (CSS has been noted to be tricky) and work on the implementation of different card views.

## [TO DO]
1. Prioritize helping the other teams with CSS as needed, and "prettify" any UI elements as needed.
2. Develop the different card views (big card view, small card view, list view).
